x/dsmr/node.go

```diff
@@ -80,25 +83,36 @@ func New[T Tx](
 			storage: storage,
 		},
 		storage: storage,
-	}, nil
+		log:     logging.NewLogger("dsmr"),
```

Comment:
let's pass the log in from the constructor instead of creating one here

Reply:
yes, of course.